vmdb/app/models/hosts_storages.rb

```diff
@@ -1,3 +1,3 @@
 class HostsStorages < ActiveRecord::Base
-  table_name = "hosts_storages"
+  self.table_name = "hosts_storages"
```

Comment:
Does it work the old way?  setters need the `self.` or they become local variables.